.github/workflows/reusable-jira-workflow.yml

```
@@ -22,12 +22,12 @@ jobs:
             }
 
             Set-JiraConfigServer -Server "${{ secrets.JIRA_URL }}"
-            if("${{ inputs.branch_name }}" -match "[a-zA-Z]+[_-][0-9]+" -eq $False)
+            if("${{ inputs.branch_name }}" -match "\/[a-zA-Z]+[_-][0-9]+" -eq $False)
```

You can use a lookbehind assertion to avoid modifying line 30

```suggestion
            if("${{ inputs.branch_name }}" -match "(?<=\/)[a-zA-Z]+[_-][0-9]+" -eq $False)
```

https://tools.meziantou.net/regex-tester?Replace=False&Pattern=%28%3F%3C%3D%5C%2F%29%5Ba-zA-Z%5D%2B%5B_-%5D%5B0-9%5D%2B&Text=feature%2Fidp-897&Options=CultureInvariant

---

Oh nice, even better then